README.md

```diff
@@ -76,6 +78,22 @@ irb(main):004:0> Docker.options
 
 Before doing anything else, ensure you have the correct version of the Docker API. To do this, run `Docker.validate_version!`. If your installed version is not supported, a `Docker::Error::VersionError` is raised.
 
+### SSL
```

Comment:
Actually, the only reason why I'm adding this is because someone wasn't using boot2docker and didn't know what to do.